opendbc/car/hyundai/interface.py

```diff
@@ -25,7 +25,7 @@ def _get_params(ret: structs.CarParams, candidate, fingerprint, car_fw, experime
     # FIXME: the Optima Hybrid 2017 uses a different SCC12 checksum
     ret.dashcamOnly = candidate in {CAR.KIA_OPTIMA_H, }
 
-    hda2 = Ecu.adas in [fw.ecu for fw in car_fw]
+    hda2 = True # Ecu.adas in [fw.ecu for fw in car_fw]
```

Comment:
Pretty sure this wasn't intentional.

Reply:
Opened PR to handle this change: https://github.com/commaai/opendbc/pull/1285